clang/lib/Index/IndexUnitWriter.cpp

```diff
@@ -281,8 +281,9 @@ void IndexUnitWriter::getUnitNameForAbsoluteOutputFile(StringRef FilePath,
   Str.push_back('-');
   // Need to be sure we use the remapped path to keep things hermetic.
   std::string RemappedPath = Remapper.remapPath(FilePath);
-  llvm::hash_code PathHashVal = llvm::hash_value(RemappedPath);
-  llvm::APInt(64, PathHashVal).toString(Str, 36, /*Signed=*/false);
+  auto PathHashVal = llvm::xxh3_64bits(RemappedPath);
+  SmallString<128> HashStr;
```

Comment:
Is `HashStr` unused?

Reply:
Indeed. I think I swapped implementations here but... do not recall. Thanks.